_pages/welcome-to-TTS/getting-started.md

```diff
@@ -244,6 +244,11 @@ Partners come to 18F because they want help with hard technical problems, often
 
 [Consulting and communicating effectively with partners]({{site.baseurl}}/how-we-relate-to-partners/) is how 18F is successful, and it's how we help our partners be successful, too. You'll also want to read up on [how we collaborate with partners]({{site.baseurl}}/how-we-collaborate/), which includes how we make remote collaboration work, how we model agile practices, and a lot more.
 
+## Diversity, accessibility, ethics and culture at TTS
+TTS strives to thoughtfully and intentionally represent the broad American populace we serve. To that end, we strongly encourage reading over the [Diversity, equity, and inclusion]({{site.baseurl}}/diversity) materials in the handbook, and consider joining the #diversity and #accessibility guilds. 
+
```

Comment:
I'd like to put more in here about a11y, but don't have a good handle on what one should know when onboarding. Suggestions welcomed.

Reply:
At least for my cohort, the developer onboarding checklist included this [udacity accessibility course](https://www.udacity.com/course/web-accessibility--ud891) which is pretty fantastic. It's definitely code heavy, but it's got a lot of great content even if you don't do the exercises.

Reply:
Given the general audience of the handbook, especially the onboarding materials, I hesitate to go to code-heavy.

Reply:
Maybe that TTS promotes inclusive design (also known as "universal design") in our work across different roles. During onboarding we explain this as "the belief and methodology of creating buildings, products, or environments to make them accessible to all people, regardless of age, disability, or other factors."

@heymatthenry yay! so glad you found that course helpful! We encourage folks to go skip the coding parts anyway.

Reply:
@Jacklynn I added a bit there.